packages/dds/test-dds-utils/src/minification.ts

```diff
@@ -190,12 +199,18 @@ export class FuzzTestMinimizer<
 				return false;
 			}
 
+			// reproduce based on the final line+col of the error
+			const message = error.stack
+				?.split("\n")
+				.map((s) => s.trim())
+				.find((s) => s.startsWith("at"));
```

Comment:
We will probably want to improve this check in the future. It solves the problem of e.g. string length being included in the error message, but introduces a new problem in that our crashes tend to come from the same assertion function.

Reply:
I debated commenting about this actually. If you want to change it now, I'd make the policy injectable, e.g.

```ts
interface DDSFuzzSuiteOptions {
    minimization: {
        skip?: boolean;
        areEquivalentErrors: (original: Error, new: Error) => boolean
    }
}
```

For the `assert` concern, might be worth special-casing it and automatically counting different error codes as different.

Reply:
I am planning to merge this now as-is, but follow up with that change. I'm dogfooding these minimization changes in next to debug obliterate reconnect, and this change being in next will let me clean up a lot of my local changes.